test/integration/test_raw_api.py

```diff
@@ -362,6 +362,23 @@ def raw_api_test_helper(raw_api, should_cleanup_old_buckets):
         server_side_encryption=sse_b2_aes,
     )
 
+    # b2_upload_file_with_cache_control
+    print('b2_upload_file_with_cache_control')
+    file_name = 'test.txt'
+    file_contents = b'hello world'
+    file_sha1 = hex_sha1_of_stream(io.BytesIO(file_contents), len(file_contents))
+    file_dict = raw_api.upload_file(
+        upload_url,
+        upload_auth_token,
+        file_name,
+        len(file_contents),
+        'text/plain',
+        file_sha1,
+        {'color': 'blue'},
+        io.BytesIO(file_contents),
+        cache_control='private, max-age=2222',
+    )
+
```

Comment:
in the list_file_versions test below we should check whether the value is saved properly and returned to us by the server in a way that we expect